src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    public static void print(String text) {
+        System.out.println("=======================================");
```

Comment:
Would it be better to extract the strings in this class (e.g. `==========`, `Congratulations on finishing this task!`) into named constants?